<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Finputdatapetugas</class>
 <widget class="QWidget" name="Finputdatapetugas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>221</width>
     <height>205</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="namaPetugasLabel">
      <property name="text">
       <string>Nama Petugas</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="tempatLahirLabel">
      <property name="text">
       <string>Tempat Lahir</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="tempatLahirLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="tanggalLahirLabel">
      <property name="text">
       <string>Tanggal Lahir</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDateEdit" name="tanggalLahirDateEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="agamaLabel">
      <property name="text">
       <string>Agama</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="agamaLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="jenisKelaminLabel">
      <property name="text">
       <string>Jenis Kelamin</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="jenisKelaminComboBox"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="jabatanLabel">
      <property name="text">
       <string>Jabatan</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="jabatanComboBox"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="alamatLabel">
      <property name="text">
       <string>Alamat</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="alamatLineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="namaPetugasLineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="kodePetugasLabel">
      <property name="text">
       <string>Kode Petugas</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="kodePetugasLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>221</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>FORM INPUT DATA PETUGAS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSimpanDataPetugas">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEditDataPetugas">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapusDataPetugas">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
